package.json

```diff
@@ -24,6 +24,10 @@
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^4.22.0",
     "@typescript-eslint/parser": "^4.22.0",
-    "eslint": "^7.24.0"
+    "eslint": "^7.24.0",
+    "jest": "^26.6.3"
+  },
+  "scripts": {
+    "test": "jest --verbose ./test"
   }
```

Comment:
I think this is duplicated with the `scripts` key above - let's just change the existing `test` to now use jest!

Reply:
Oh yep, didn't see it before. Thanks.